stdlib/mlang/postprocess.mc

```diff
@@ -0,0 +1,61 @@
+-- This transformation converts the `Name`s of semantic functions to `Name`s 
+-- with the same symbol, but also have a unique string in the shape of 
+-- <LangName>_<SemName>. 
+-- 
+-- This transformation is added because some parts of the exising MExpr
+-- transformations to OCaml do not respect the symbols fully and also require
+-- the strings of names to be unique. 
```

Comment:
This sounds rather worrying, do you have pointers to what doesn't handle things correctly?

Reply:
I discussed this a little bit with @larshum and it seems that the utest-generation sometimes uses `nameGetStr` for the names of generated code which can cause incorrect bheaviour when you have names with the same strings but different symbols.

Reply:
Correct. I think those cases should be fairly easy to fix, by adding the symbol to the string we generate. There's also code in Miking DPPL that assumes the MLang name mangling works in a particular way, which would surely break as well.

Reply:
I have created an issue for this (see #850). The MLang pipeline will just use the `postprocess` work around until this issue is resolved.